scripts/test

```diff
@@ -37,7 +43,12 @@ copy_logs(){
 run_tests()
 {
     pushd ./tests/validation
-    tox || [ `uname` == "Linux" ] && copy_logs && exit 1
+    if [ "${CATTLE_TEST_PARALLEL_THREADS}" -gt 1 ] 
```

Comment:
Could we _just_ have this line, as opposed to the if/else:
```
tox -- -n "${CATTLE_TEST_PARALLEL_THREADS}" || [ `uname` == "Linux" ] && copy_logs && exit 1
```
...since CATTLE_TEST_PARALLEL_THREADS=1 would work?
Or did you not do that because it changes the output format?